Hymns/Tags/SelectLabelView.swift

```diff
@@ -0,0 +1,107 @@
+import SwiftUI
+
+struct SelectLabelView: View {
+    @Binding var tagColor: TagColor
+    @State var blueButton = false
+    @State var greenButton = false
+    @State var yellowButton = false
+    @State var redButton = false
```

Comment:
what if you did this:

`@State colorSelected: TagColor = .none`

Then when you set the backgrounds:

`.stroke(Color.blue, lineWidth: self.colorSelected == .blue ? 3 : 0)`

Reply:
fixed for you. feel free to resolve if it makes sense.

Reply:
I changed the stroke color from always being blue. But other than that this is great!